airflow/migrations/versions/2c6edca13270_resource_based_permissions.py

```diff
@@ -319,3 +344,7 @@ def upgrade():
 
 def downgrade():
     """Unapply Resource based permissions."""
+    log = logging.getLogger()
+    handlers = log.handlers[:]
```

Comment:
Worth adding a note why this is needed?

Reply:
I think this logging is not necessary, however, I copied it from the upgrade function